src/shared/VirtualStreamingJanus.js

```diff
@@ -456,6 +467,14 @@ export default class VirtualStreamingJanus {
         let stream = new MediaStream();
         stream.addTrack(track.clone());
         this.trlAudioMediaStream = stream;
+
+        // Connect audio stream to analyser node (to extract volume).
+        if (this.trlAudioContext) {
+          this.trlAudioContext.close();
+          this.trlAudioContext = null;
+        }
+        this.trlAudioContext = new AudioStreamVolume(this.trlAudioMediaStream);
```

Comment:
I think we need here to check browser as well. Because on Chrome i'm get error here. If we on Chrome does not use AudioStreamVolume, so why we init it here?

Reply:
Fixed, please check again.